src/base/static/js/views/app-view.js

```diff
@@ -695,24 +723,44 @@ module.exports = Backbone.View.extend({
   newPlace: function() {
     var self = this;
 
+    // REACT PORT SECTION //////////////////////////////////////////////////////
     if (!this.placeFormView) {
-      this.placeFormView = new PlaceFormView({
-        appView: this,
-        router: this.options.router,
-        placeConfig: this.options.placeConfig,
-        mapConfig: this.options.mapConfig,
-        userToken: this.options.userToken,
-        geometryEditorView: this.mapView.geometryEditorView,
-        collectionsSet: {
-          places: this.places,
-          landmarks: this.landmarks,
-        },
-      });
+      // this.placeFormView = new PlaceFormView({
+      //   appView: this,
+      //   router: this.options.router,
+      //   placeConfig: this.options.placeConfig,
+      //   mapConfig: this.options.mapConfig,
+      //   userToken: this.options.userToken,
+      //   geometryEditorView: this.mapView.geometryEditorView,
+      //   collectionsSet: {
+      //     places: this.places,
+      //     landmarks: this.landmarks,
+      //   },
+      // });
```

Comment:
We can probably delete this code, instead of commenting it out?

Reply:
yes, I'll clean up all the commented stuff before finalizing everything